app/models/rental.py

```diff
@@ -0,0 +1,39 @@
+from app import db
+import datetime
+from flask import current_app
+from sqlalchemy import Table, Column, Integer, ForeignKey
```

Comment:
since we are importing db, we can access `Column, ForeignKey` etc through it instead and don't need this

```suggestion

```